app/models/prescription_drug.rb

```diff
@@ -13,6 +13,13 @@ class PrescriptionDrug < ApplicationRecord
   validates :is_protocol_drug, inclusion: {in: [true, false]}
   validates :is_deleted, inclusion: {in: [true, false]}
 
+  def self.prescribed_as_on(date: Date.today)
```

Comment:
```suggestion
  def self.prescribed_as_on(date: Date.current)
```

Reply:
Given the method naming, I don't think it makes sense to provide a default argument. I'd argue this method should be passed a date always.

PS, does the method behave gracefully if we pass it a DateTime as well?